puppet/modules/jenkins_job_builder/files/config/scripts/gemset_cleanup.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+ruby=2.0.0
```

Comment:
I think it *can* be made sufficiently flexible, but for our first pass we may just want to use multiple publishers. We can then verify the xml doesn't change (or changes suitably) when we refactor.